@xen-orchestra/lite/src/components/pool/system/PoolStorageConfiguration.vue

```diff
@@ -0,0 +1,91 @@
+    <UiTitle>
+      {{ $t('storage-configuration') }}
+    </UiTitle>
+    <VtsLoadingHero v-if="!isReady" type="card" :title="$t('storage-configuration')" />
+    <template v-else>
+      <VtsQuickInfoRow :label="$t('default-storage-repository')">
+        <template #value>
+          <UiLink v-if="defaultSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ defaultSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('suspend-storage-repository')">
+        <template #value>
+          <UiLink v-if="suspendSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ suspendSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('crash-dump-storage-repository')">
+        <template #value>
+          <UiLink v-if="crashDumpSr" v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+            {{ crashDumpSr.name_label }}
+          </UiLink>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+      <VtsQuickInfoRow :label="$t('heartbeat-storage-repository')">
+        <template #value>
+          <ul v-if="haSrs !== undefined && haSrs.length > 0">
+            <li v-for="haSr in haSrs" :key="haSr.uuid">
+              <UiLink v-tooltip="$t('coming-soon')" :icon="faDatabase" size="medium">
+                {{ haSr.name_label }}
+              </UiLink>
+            </li>
+          </ul>
+          <template v-else>
+            {{ $t('none') }}
+          </template>
+        </template>
+      </VtsQuickInfoRow>
+    </template>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XenApiPool, XenApiSr } from '@/libs/xen-api/xen-api.types'
+import { useSrStore } from '@/stores/xen-api/sr.store'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import VtsLoadingHero from '@core/components/state-hero/VtsLoadingHero.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiLink from '@core/components/ui/link/UiLink.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { vTooltip } from '@core/directives/tooltip.directive'
+import { faDatabase } from '@fortawesome/free-solid-svg-icons'
+import { computed } from 'vue'
+
+const { pool } = defineProps<{
+  pool: XenApiPool
+}>()
+
+const { getByOpaqueRef, isReady } = useSrStore().subscribe()
+
+const defaultSr = computed(() => (pool.default_SR ? getByOpaqueRef(pool.default_SR) : undefined))
+const suspendSr = computed(() => (pool.suspendSr ? getByOpaqueRef(pool.suspendSr) : undefined))
+const crashDumpSr = computed(() => (pool.crash_dump_SR ? getByOpaqueRef(pool.crash_dump_SR) : undefined))
```

Comment:
From what I see in the `XenApiPool` type, these properties are always defined (even if the value can be `OpaqueRef:NULL`). See [@vates/types/src/xen-api.mts](https://github.com/vatesfr/xen-orchestra/blob/77cb05d8619cf4d7b8afe6217e5aa2efd45e0e02/%40vates/types/src/xen-api.mts#L176) for example.
```suggestion
const defaultSr = computed(() => (pool.default_SR !== 'OpaqueRef:NULL' ? getByOpaqueRef(pool.default_SR) : undefined))
const suspendSr = computed(() =>
  pool.suspend_image_SR !== 'OpaqueRef:NULL' ? getByOpaqueRef(pool.suspend_image_SR) : undefined
)
const crashDumpSr = computed(() =>
  pool.crash_dump_SR !== 'OpaqueRef:NULL' ? getByOpaqueRef(pool.crash_dump_SR) : undefined
)
```